stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -172,6 +203,15 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
         }
     }
 
+    private void applyTint() {
```

Comment:
At the moment, yes, because the known cards are full PNGs that are colored - so if you apply tint to them, it blacks them out.